### DISCUSSION TITLE
GoPro Cloud during long timelapse

### DISCUSSION THREAD

**Discussion by jarobaina** [gopro/labs#1034] [Q&A]
Hi guys, I will soon be doing a time-lapse of the handmade construction of a small sailboat. The construction is estimated to take approximately 90 days. I have
almost figured out the camera and power supply settings for this period, but I would like to know if the camera will sync the photos taken with the GoPro cloud since I will have a wifi network accompanying the camera if I can achieve this.
Thanks.

**Comment by dnewman-gpsw** [comment]
Maybe, it would take some experimentation to see if the network is reliable enough. You can script uploads, but the reliability is reduced then uploading, so you would have to check on the camera every week.

**Comment by dnewman-gpsw** [comment]
Try https://gopro.github.io/labs/control/dailytl/

**Reply by jarobaina** [reply]
Thank you so much, I've been looking through all the "preconfigured" options from Gopro Labs but I couldn't find this one in particular. I think it's exactly what I need, even with a daily start and end time. I'll test it for a few days. Thanks
